extra/grafana/grafana-purefb-flashblade-overview.json

```diff
@@ -61,7 +61,7 @@
       }
     ]
   },
-  "description": "v1.0.5 Fixes a bug with Grafana setting datasource \"uid\": \"${DS_PROMETHEUS}\" causing some user panels not to successfully load. Fix involves manually overriding with \"uid\": \"$datasource\".",
+  "description": "v1.0.6 Fixes a bug with Grafana setting datasource \"uid\": \"${DS_PROMETHEUS}\" causing some user panels not to successfully load. Fix involves manually overriding with \"uid\": \"$datasource\".",
```

Comment:
@paulojmdias I see you changed the version, please can you just change the entire description to summerises the changes in this version. Thanks.
```
"description": "v1.0.6 Fix: Changes Average IOPS by IO Type and array(s) from Instant to Range to calculate average over range rather returning a single average value.",
```

Reply:
Sorry, my mistake @james-laing. Fixed ✅